fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/RecordCursorResult.java

```diff
@@ -200,6 +200,14 @@ public int hashCode() {
         }
     }
 
+    /**
+     * Return whether or not this result does not include a next value but include a continuation (i.e. source is not exhausted).
```

Comment:
This sentence is a little hard to parse.  How about something like:

    Returns true if the cursor has reached its end but a continuation is still available (i.e. the source is not yet exhausted)

Reply:
Done.